### DISCUSSION TITLE
Linux icon with custom layout

### DISCUSSION THREAD

**Discussion by emm1R** [wailsapp/wails#2400] [Q&A] [answered]
The appicon.png won't work for Linux unless I embed it in main.go. However, I cannot embed the image from build/ because the project has a similar layout as https://github.com/wailsapp/wails/tree/master/v2/examples/customlayout. Embed does not allow `../` in the path. So is there a way for make this work without having two copies of appicon.png?

**Comment by leaanthony** [comment, marked as answer]
Simple Go trick: put a `.go` file next to the icon and expose it using a capital first letter. Import the package and use the embedded icon. We use this trick here: 
https://github.com/wailsapp/wails/blob/master/v2/internal/frontend/runtime/runtime_dev_desktop.go

**Reply by emm1R** [reply]
Oh, right. Like the assets.
